src/components/molecules/ZopaFooter/Links/Links.tsx

```diff
@@ -136,24 +140,24 @@ const linkGroups = [
     heading: 'Navigation',
     links: [
       {
-        href: 'https://www.zopa.com/contact',
+        href: `${baseUrl}/contact`,
         label: 'Support',
       },
       {
-        href: 'https://www.zopa.com/support/faqs',
+        href: `${baseUrl}/support/faqs`,
         label: 'Common Questions',
       },
       {
-        href: 'https://www.zopa.com/sitemap',
+        href: `${baseUrl}/sitemap`,
         label: 'Sitemap',
       },
     ],
   },
 ];
 
-const Links = () => (
+const Links: FC<ILinksProps> = ({ baseUrl = 'https://www.zopa.com' }) => (
```

Comment:
no need to use `FC` here, since it would mean that Links can accept `children` which they don't.